src/libraries/Fuzzing/DotnetFuzzing/DotnetFuzzing.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="SharpFuzz" Version="2.1.1" />
+    <PackageReference Include="System.Formats.Nrbf" Version="9.0.0-preview.7.24405.7" />
```

Comment:
I am not sure if this is the right way to do it (I am afraid it will simply use this particular version all the time, rather than what has been built locally).

@MihaZupan @ericstj what is the best way to reference a project that produces an OOB package?

Reply:
> I am afraid it will simply use this particular version all the time, rather than what has been built locally

I believe so. We'll have to fix #107204 to make this work beyond one-off tests like this.

Reply:
Though your workaround from https://github.com/dotnet/runtime/pull/107368#issuecomment-2330930150 would likely work for the automated infra until then
> `<HintPath>..\..\..\..\artifacts\bin\System.Formats.Nrbf\Debug\net8.0\System.Formats.Nrbf.dll</HintPath>`

Reply:
> Curious if you tried a normal ProjectReference and what happened. We've done a lot of work to make our OOB packages look like normal projects, I think it may just work.

Just tried, it works! Thank you!